.github/workflows/bc.yml

```diff
@@ -25,7 +25,7 @@ name: backwards compatibility
 
 jobs:
   roave_bc_check:
-    uses: yiisoft/actions/.github/workflows/bc.yml@master
+    uses: yiisoft/actions/.github/workflows/bc.yml@48ade4ccb74825ff7f725afc17591572ecbe71c5
```

Comment:
Should we revert that in yiisoft/actions?

Reply:
Please do the revert.

Reply:
https://github.com/yiisoft/actions/pull/56